graphgym/custom_graphgym/act/example.py

```diff
@@ -18,5 +20,5 @@ def forward(self, x):
             return x * torch.sigmoid(x)
 
 
-register_act('swish', SWISH(inplace=cfg.mem.inplace))
-register_act('lrelu_03', nn.LeakyReLU(0.3, inplace=cfg.mem.inplace))
+register_act('swish', partial(SWISH, inplace=cfg.mem.inplace))
```

Comment:
```suggestion
register_act('swish', lambda: SWISH(inplace=cfg.mem.inplace))
```
WDYT? No import needed.

Reply:
I actually like using `partial` here :)

Reply:
Since those are examples, maybe it is best to register one activation with `partial` and one using `lambda`. This way, the people looking at the examples have more options to choose from.

Reply:
I think its fine just to use lambda. LGTM